pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1766,3 +1788,37 @@ func getServingCertsMode(remoteKubernetes *ispnutil.Kubernetes) string {
 	}
 	return ""
 }
+
+func (r *ReconcileInfinispan) applyLabelsToCoordinatorsPod(podList *corev1.PodList, infinispan *infinispanv1.Infinispan, cluster ispnutil.ClusterInterface, logger logr.Logger) (bool, error) {
+	pass, err := cluster.GetPassword(consts.DefaultOperatorUser, infinispan.GetSecretName(), infinispan.GetNamespace())
+	coordinatorFound := false
+	if err != nil {
+		return coordinatorFound, err
+	}
+	protocol := string(infinispan.GetEndpointScheme())
+	for _, item := range podList.Items {
+		cacheManagerInfo, err := cluster.GetCacheManagerInfo(consts.DefaultOperatorUser, pass, "default", item.Name, item.Namespace, protocol)
```

Comment:
@rigazilla it would be nice to place "default" CacheManager to the constants. Do we need to have it as parameter? Is it possible to another manager rather than "default"

Reply:
I don't know.
@pruivo you know if we have to care about CacheManager != "default"?

Reply:
Not atm, I'll create a constant for `default` for now